### PR TITLE
btc 0.21.0 → 22.0

### PR DESCRIPTION
**Pull request by Dehumanizer77**

### CODE REVIEW

File: configs/coins/bitcoin_testnet.json [hunk 1]
```diff
@@ -22,10 +22,11 @@
     "package_name": "backend-bitcoin-testnet",
     "package_revision": "satoshilabs-1",
     "system_user": "bitcoin",
-    "version": "0.21.1",
-    "binary_url": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/bitcoin-0.21.1-x86_64-linux-gnu.tar.gz",
-    "verification_type": "gpg-sha256",
-    "verification_source": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/SHA256SUMS.asc",
+    "system_user": "bitcoin",
```
Comment (gruve-p) [inline configs/coins/bitcoin_testnet.json:25]: 2 times `"system_user": "bitcoin",`

File: configs/coins/bitcoin_signet.json [hunk 2]
```diff
@@ -22,10 +22,11 @@
     "package_name": "backend-bitcoin-signet",
     "package_revision": "satoshilabs-1",
     "system_user": "bitcoin",
-    "version": "0.21.1",
-    "binary_url": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/bitcoin-0.21.1-x86_64-linux-gnu.tar.gz",
-    "verification_type": "gpg-sha256",
-    "verification_source": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/SHA256SUMS.asc",
+    "system_user": "bitcoin",
```
Comment (gruve-p) [inline configs/coins/bitcoin_signet.json:25]: 2 times `"system_user": "bitcoin",`

File: configs/coins/bitcoin.json [hunk 3]
```diff
@@ -22,10 +22,11 @@
     "package_name": "backend-bitcoin",
     "package_revision": "satoshilabs-1",
     "system_user": "bitcoin",
-    "version": "0.21.1",
-    "binary_url": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/bitcoin-0.21.1-x86_64-linux-gnu.tar.gz",
-    "verification_type": "gpg-sha256",
-    "verification_source": "https://bitcoincore.org/bin/bitcoin-core-0.21.1/SHA256SUMS.asc",
+    "system_user": "bitcoin",
```
Comment (gruve-p) [inline configs/coins/bitcoin.json:25]: 2 times `"system_user": "bitcoin",`

**Comment by Dehumanizer77** [comment]
Thanks, fixed

**Pull request merged**